settings/authentication-personalization/authentication-setup/mintlify.mdx

```diff
@@ -1,35 +1,44 @@
 ---
 title: 'Mintlify Dashboard Handshake'
-description: 'Use the Mintlify Dashboard to authenticate users'
+description: 'Use Mintlify to authenticate users'
 ---
 
+<Note>
+Mintlify Dashboard Handshake is available only by request. Please <a href="mailto:sales@mintlify.com">contact our sales team</a> for access. After we've enabled access, you can follow the implementation instructions.
+</Note>
+
+
+You can use Mintlify to authenticate and manage access to your documentation. Anyone that can access
+your dashboard will automatically be able to see your documentation.
+
+This handshake method also enables private preview deployments, so that only Mintlify authenticated users will be able to access your preview deployments. 
```

Comment:
This is a bit misleading - you do not need to use the dashboard handshake for your main handshake method to use it for your preview deployments. Maybe something like this?

"This handshake method can also be used to protect preview deployments, so that only Mintlify authenticated users will be able to access your preview deployments. You can enable private preview deployments regardless of the type of authentication used for your docs"

or something like that

Reply:
agreed - it is 2 different toggles in the UI and 2 different features.